templates/CentOS-7.6-x86_64.ks.erb

```diff
@@ -0,0 +1,109 @@
+#platform=x86, AMD64, or Intel EM64T
+#version=DEVEL
+# Firewall configuration
+firewall --disabled
+# Install OS instead of upgrade
+install
+# Use network installation
+url --url="http://<%= @server_ip %>/distros/<%= @distro %>"
+# Root password
+rootpw --iscrypted <%= @defaultrootpw %>
+# System authorization information
+auth  --useshadow  --passalgo=sha512 --enablefingerprint
+# Use text mode install
+text
+# System keyboard
+keyboard us
+# System language
+lang en_US
+# SELinux configuration
+selinux --disabled
+# Do not configure the X Window System
+skipx
+# Installation logging level
+logging --level=info
+# create yum repos for installation
+$yum_repo_stanza
+# Reboot after installation
+reboot
+# System timezone
+timezone  Europe/Zagreb
+# Network information
+network  --bootproto=dhcp --device=eth0 --onboot=on
+# System bootloader configuration
+bootloader --location=mbr --driveorder=sda
+# Clear the Master Boot Record
+zerombr
+# Partition clearing information
+clearpart --drives=sda --all
+# Disk partitioning information
+part /boot --ondisk=sda --asprimary --fstype="ext2" --size=100
+part swap  --ondisk=sda --asprimary --fstype="swap" --size=4096
+part /     --ondisk=sda --asprimary --fstype="ext3" --size=20480
+part /data --ondisk=sda --asprimary --fstype="ext4" --size=4096 --grow
+
+#
+# pre script
+#
+%pre
+
+# log pre
+$SNIPPET('log_ks_pre')
+$SNIPPET('kickstart_start')
+# network config
+$SNIPPET('pre_install_network_config')
+# enable installation monitoring
+$SNIPPET('pre_anamon')
+
+#
+# packages list
+#
+%packages
+
+@base
```

Comment:
Can you remove `@base`, that would eliminate the need to disable all those services?

This is kinda minimal install for CentOS 7:

```@core
net-tools
xfsprogs
-firewalld
-firewalld-filesystem
-alsa-firmware
-alsa-tools-firmware
-iwl*firmware
-ivtv-firmware
-aic49xx-firmware
-crystalhd-firmware
-plymouth*
-wpa_supplicant
-NetworkManager
-NetworkManager-team
-NetworkManager-tui
-NetworkManager-wifi
```

Reply:
fine by me